moshi/src/test/java/com/squareup/moshi/JsonReaderTest.java

```diff
@@ -542,36 +545,37 @@ private void assertNotANumber(String s) throws IOException {
     JsonReader reader = newReader("[12.34e5x]");
     reader.setLenient(true);
     reader.beginArray();
-    assertEquals(STRING, reader.peek());
+    assertThat(reader.peek()).isEqualTo(STRING);
     try {
       reader.nextInt();
       fail();
     } catch (IllegalStateException expected) {
     }
-    assertEquals("12.34e5x", reader.nextString());
+    assertThat(reader.nextString()).isEqualTo("12.34e5x");
   }
 
   @Test public void peekLongMinValue() throws IOException {
     JsonReader reader = newReader("[-9223372036854775808]");
     reader.setLenient(true);
     reader.beginArray();
-    assertEquals(NUMBER, reader.peek());
-    assertEquals(-9223372036854775808L, reader.nextLong());
+    assertThat(reader.peek()).isEqualTo(NUMBER);
+    Object expected = -9223372036854775808L;
```

Comment:
I made this by refactoring by making my own Assert class, and then hitting inline on everything. IntelliJ extracted this one.